### PR TITLE
COMMERCE-12396

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-12396

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 1]
```diff
@@ -440,6 +440,224 @@ definition {
...
+
+			CommerceAcceleratorsInitializer.gotoMiniumProductDetailsPage(productName = "Brake Fluid");
+		}
+
+		task ("Then assert that MIN93016A(pre-selected sku) has only UOM1(UOM field not clickable)") {
+			AssertSelectedLabel(
+				locator1 = "CommerceFrontStore#FS_PRODUCT_UOM_SELECT",
+				value1 = "UOM1");
+
+			AssertElementPresent(
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:522]: Add a small for for checking if the UOM2, UOM3, UOM4 are not present

Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 2]
```diff
@@ -440,6 +440,224 @@ definition {
...
+				var index = StringUtil.extractFirst(${indexAndUOMName}, ":");
+				var uomName = StringUtil.extractLast(${indexAndUOMName}, ":");
+
+				AssertElementPresent(
+					index = ${index},
+					locator1 = "CommerceFrontStore#FS_PRODUCT_UOM_SELECTOR_VALUE_BY_INDEX",
+					uomName = ${uomName});
+			}
+
+			AssertElementNotPresent(
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:579]: Add a small for for checking if the UOM1 is not present

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 3]
```diff
@@ -440,6 +440,224 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12396. As admin, I can add to the cart a multi SKUs product with UOM."
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:443]: As a buyer

**Review by pietrobocale — CHANGES_REQUESTED** [review]
Review Completed check the comments

Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
New PR is here:

https://github.com/liferay-commerce/liferay-portal/pull/4310

**Pull request closed without merging**